src/tree/ResourceGroupTreeItem.ts

```diff
@@ -75,9 +77,17 @@ export class ResourceGroupTreeItem extends GroupTreeItemBase {
     }
 
     public async deleteTreeItemImpl(context: IActionContext): Promise<void> {
-        const client: ResourceManagementClient = await createResourceClient([context, this]);
-        await client.resourceGroups.beginDeleteAndWait(this.name);
-        ext.outputChannel.appendLog(localize('deletedRg', 'Successfully deleted resource group "{0}".', this.name));
+        const wizard = new AzureWizard<DeleteResourceGroupContext>({
```

Comment:
It sorta seems like overkill to use the full wizard framework to do the delete. Is the experience significantly different if we just use a simple `ExecuteActivity` to wrap the create RG client+delete RG?

Reply:
It may be overkill, but using a full AzureWizard automatically handles showing the progress, and takes into account the setting we added to suppress the progress notifications.

I could write a utility function to make creating azure wizards easier.

Reply:
In a multi-step process I think it definitely makes sense to use the wizard and take advantage of its progress reporting, but this is just a single-step delete.

Reply:
Ok, I overlooked this originally so I'll refactor `ExecuteActivity` a bit and also expose it in utils so we can use it outside of the AzureWizard.

Reply:
Talked offline and decided it's ok to use a AzureWizard in this case